new_data_contribution.md

```diff
@@ -0,0 +1,274 @@
+<!--
+
+author:   Arcus Library Sciences
+email: dlarcuslibraryscience@chop.edu
+version:  1.0.0
+current_version_description: Brief description of why this version exists
+module_type: standard
+docs_version: 1.1.0
+language: en
+narrator: UK English Female
+title: Arcus Data Contribution Orientation
+comment:  Learn about archiving research data with Arcus
+long_description: Arcus Archives is the canonical repository for research data at CHOP. Archiving research data preserves the important research performed at CHOP according to archival standards while facilitating data sharing. This module reviews reasons to archive research data at Arcus, and the data scoping, privacy review and technical considerations taken before receiving data. Module 2 covers the steps of archiving the data at Arcus.
+
+estimated_time: 30 minutes
+
+@pre_reqs: It's helpful to have reviewed the [Arcus website](https://arcus.chop.edu) at  (available only on the CHOP network), to understand Arcus’s overall goals.
```

Comment:
```suggestion
@pre_reqs
It's helpful to have reviewed the [Arcus website](https://arcus.chop.edu) at  (available only on the CHOP network), to understand Arcus’s overall goals.
```

This is what's called a "block macro" in liascript, so the text goes between the @pre_reqs and @end tags rather than after a colon. The lone @end after this macro was causing the error that then prevented the next macro (learning_objectives) from displaying properly.